### PR TITLE
PUSHT / PUSHF

### PR DESCRIPTION
**Pull request by Hecate2**

### CODE REVIEW

**Comment by Hecate2** [comment]
@dotnet-policy-service agree

File: docs/en-us/reference/neo_vm.md [hunk 1]
```diff
@@ -50,6 +50,14 @@ The constant instructions mainly complete the function of pushing constants or a
...
 | Function   | Push an integer onto the stack, the bit length of which is specified with the number 8\16\32\64\128\256. |
 
+#### PUSHT/PUSHF
+
+| 指令   | PUSHT, PUSHF                                      |
+| ------ | ------------------------------------------------- |
+| 字节码 | 0x08, 0x09                                        |
+| 系统费 | 0.00000001 GAS                                    |
+| 功能   | Push the Boolean value True/False onto the stack. |
+
```
Comment (superboyiii) [inline docs/en-us/reference/neo_vm.md:60]:
```suggestion
#### PUSHT/PUSHF

| 指令   | PUSHT, PUSHF                                      |
| ------ | ------------------------------------------------- |
| Bytecode | 0x08, 0x09                                        |
| Fee | 0.00000001 GAS                                    |
| Function | Push the Boolean value True/False onto the stack. |

```

Comment (Hecate2) [reply docs/en-us/reference/neo_vm.md:60]: Oh thanks

**Comment by Celia18305** [comment]
Fixed in https://github.com/neo-project/neo-dev-portal/pull/69 and changes will be reflected in the new neo dev portal: https://developers.neo.org/docs/

**Pull request closed without merging**